facts/inventory.py

```diff
@@ -73,7 +73,7 @@ def genvlans(line, building):
     elems = re.split(r"\t+", line)
     rangeb, rangee = elems[2].split("-")
     for i in range(int(rangeb), int(rangee) + 1):
-        ipv6prefix = elems[3].split("/")[0].split(rangeb + "::")[0] + str(i) + "::"
```

Comment:
This was to match the changes in VVRNG from https://github.com/socallinuxexpo/scale-network/pull/661/files#diff-077a6c63504cfb184eac78e4bd46b53e79390027b942f021589bfda90a79e1abR21

Reply:
Makes sense.